release.sh

```diff
@@ -27,126 +27,29 @@ check_git_branch() {
 
 check_git_branch
 
-# update to the latest
-git pull origin master
+sed -i -e "s/version = '.*';$/version = '$VERSION_NO_V';/" src/0_config.js
+sed -i -e "s/version = '.*';$/version = '$VERSION_NO_V';/" test/web-config.js
 
-read -p "Update CHANGELOG.md?" -n 1 -r
-echo
-if [[ $REPLY =~ ^[Yy]$ ]]
-then
-  vi CHANGELOG.md
-  git commit -am "Updated CHANGELOG.md"
-fi
-
-echo "Building files"
-
-read -p "Update 0_config.js? " -n 1 -r
-echo
-if [[ $REPLY =~ ^[Yy]$ ]]
-then
-  sed -i -e "s/version = '.*';$/version = '$VERSION_NO_V';/" src/0_config.js
-  sed -i -e "s/version = '.*';$/version = '$VERSION_NO_V';/" test/web-config.js
-fi
-
-read -p "Update package.json? " -n 1 -r
```

Comment:
happens earlier in the gha, before npm ci. I think we need to do that because we need the package.json updated before we run npm ci